### ISSUE TITLE
Deck Name Issue

### ISSUE THREAD

**Issue by Elitesparkle** [HearthSim/Hearthstone-Deck-Tracker#2375]
Sometimes the deck name on the overlay is different from the deck name in the tracker window.
I think the problem is related to renaming, editing and/or cloning a deck.

You need to check how you manage the Name field in PlayerDecks.xml.

**Comment by Epix37** [comment]
Can you be more specific than "sometimes"?

**Comment by Elitesparkle** [comment]
Some decks have that issue, and it happens every time they load on the overlay or when I export them in game.

**Comment by Epix37** [comment]
What kind of decks specifically

**Comment by Elitesparkle** [comment]
A Constructed deck that has 3 minor versions and, if I don't remember wrong, the first version has been created using the cloning function from the last version of another deck that I've then deleted.

I named it in a different way when I created it and then edited the name.
The original name is used on the overlay and when exporting it.
The new name is used in the tracker.

If I edit the name, the original name doesn't change.
